campaign/sample/lore.txt

```diff
@@ -1,17 +1,393 @@
-As for New Farsell, well, the first time I played that mission at the time, I had that eerie feeling: all those heavy defences, and yet it felt so empty...  I did take some inspiration from an unsettling passage in the novel John Dies At The End (which I also recommend to any fans of fantastic/weird genre), with a slightly different form."
+------ Commander-centric combat ------
+
+Commanders are the most famous weapon system in use by galactic militaries. Generally, the name refers to either the insertion unit or its pilot, though strictly speaking it refers to both as well as the support system, typically a carrier starship or a surface installation.
+The support system includes the insertion method. The most common one is surface teleportation, though exoatmospheric teleportation followed by fast atmospheric insertion is also used.
+Other methods are not unheard of, but are generally special cases or, rarely, someone trying to be sneaky. This often gets Commanders destroyed, but the payoff can be big. In theory at least.
+
+It is generally impossible to heavily defend the entire surface of a planet. So there will always be remote spots where a Commander can be sent, and quickly build up an army.
+If no suitable spot can be found, or those who can be found are deemed inadequate, one can be created with heavy use of long-range fires to destroy thin local defenses and/or insertion jammers. However, this is typically extremely costly as it requires overwhelming enemy aerospace defenses, and as such is avoided when possible.
+
+Commander insertion can only be done with either an already established foothold on the target world, or with heavy warship support from space, or with a foothold on a nearby celestial body if one exists that is within teleportation range.
+Attacking a world without having local supporters already establishing a foothold is difficult and requires considerable orbital forces, but concentration of forces can allow great powers to overwhelm individual planetary defenses relatively easily. In theory at least.
+
+Note that outside extreme concentration of space forces, the anti-space defenses of a typical planet can easily keep attackers at bay and prevent them from direct assault. And even with extreme concentration of force, such assault would be extremely costly.
+Hence why inserting Commanders to build bridgeheads is needed instead.
+
+It is easier to neutralize long-range fires than to launch them, thus attackers and defenders generally neutralize each-other�s fires and the Commanders on the surface don�t have fire support they don�t build themselves.
```

Comment:
something is going on here with the `'` character

Reply:
Oops, fixed